Several minor UI/UX improvements

danielchalmers opened the pull request:
Shell:
 - Added menu separators
 - Added access key for "Syncthing Console" menu item
 - Added minimum width and height
 - Increased size of console buttons and make them consistent

Tray icon:
 - Added menu separators
 - Added access key for "Close to Tray" menu item
 - Added tooltip

About window:
 - Added close on escape hotkey
 - Disabled resize

Settings window:
 - Disabled resize

Conflict Resolver window:
 - Added close on enter and escape hotkeys
 - Added minimum width and height

Other:
- Change "Kill all Syncthing processes" message box image from question to warning (https://msdn.microsoft.com/en-us/library/windows/desktop/dn742473(v=vs.85).aspx)

canton7 commented on src/SyncTrayzor/Pages/ConsoleView.xaml:
```diff
@@ -15,10 +15,10 @@
-            <Button DockPanel.Dock="Right" Padding="3,0" Margin="0,3,5,3" Height="NaN" Command="{s:Action ClearLog}" Content="{l:Loc ConsoleView_ClearButton}"/>
-            <Button DockPanel.Dock="Right" Padding="3,0" Margin="0,3,5,3" Height="NaN" Command="{s:Action PauseLog}" Content="{l:Loc ConsoleView_PauseButton}" Visibility="{Binding LogPaused, Converter={x:Static s:BoolToVisibilityConverter.InverseInstance}}"/>
-            <Button DockPanel.Dock="Right" Padding="3,0" Margin="0,3,5,3" Height="NaN" Command="{s:Action ResumeLog}" Content="{l:Loc ConsoleView_ResumeButton}" Visibility="{Binding LogPaused, Converter={x:Static s:BoolToVisibilityConverter.Instance}}"/>
+            <Button DockPanel.Dock="Right" Padding="3,0" Margin="0,3,5,3" Height="NaN" Width="50" Command="{s:Action ShowSettings}" Content="{l:Loc ConsoleView_SettingsButton}"/>
```
I'm not too sure about putting widths on things holding localizable content - what if a language has a longer string here?

canton7 replied:
I think a better approach might be to specify a MinWidth, but let it expand out to infinity if it needs to.

canton7 commented on src/SyncTrayzor/Pages/Settings/SettingsView.xaml:
```diff
@@ -15,6 +15,7 @@
         MinWidth="430" MaxWidth="430"
         MinHeight="400" 
         SizeToContent="WidthAndHeight"
+        ResizeMode="CanMinimize"
```
Allowing users to make this window shorter is intentional - it's annoyingly tall on smaller screens, and it scales nicely. I have a faint memory that the NoSizeBelowScreenBehaviour below may require this behaviour as well - I'd need to check.

canton7 commented on src/SyncTrayzor/Pages/AboutView.xaml:
```diff
@@ -6,6 +6,7 @@
         Height="400" Width="250"
         WindowStyle="None" SizeToContent="Height"
         ShowInTaskbar="False"
+        ResizeMode="CanMinimize"
```
Given that WindowStyle is None, and ShowInTaskbar is False, is there much point in using CanMinimize over NoResize?

danielchalmers replied:
They have the same result AFAIK, but having it on CanMinimize makes it easier to switch to a bordered window in the future if wanted.

canton7 replied:
I think CanMinimize is misleading here - it implies that the window can be minimized when, due to the other properties on it, it can't be.

danielchalmers replied:
That's fair. Changed.

canton7 commented on src/SyncTrayzor/Pages/Settings/SettingsView.xaml:
```diff
@@ -15,7 +15,6 @@
         MinWidth="430" MaxWidth="430"
         MinHeight="400" 
         SizeToContent="WidthAndHeight"
-        ResizeMode="CanMinimize"
```
I agree with making it non-resizable, but use NoResize instead of CanMinimize.

canton7 commented:
Cheers! I'll pull it down and take a final look when I get time, hopefully either tonight or tomorrow lunchtime.

canton7 commented on src/SyncTrayzor/Pages/ConsoleView.xaml:
```diff
@@ -15,10 +15,10 @@
-            <Button DockPanel.Dock="Right" Padding="3,0" Margin="0,3,5,3" Height="NaN" Command="{s:Action ClearLog}" Content="{l:Loc ConsoleView_ClearButton}"/>
-            <Button DockPanel.Dock="Right" Padding="3,0" Margin="0,3,5,3" Height="NaN" Command="{s:Action PauseLog}" Content="{l:Loc ConsoleView_PauseButton}" Visibility="{Binding LogPaused, Converter={x:Static s:BoolToVisibilityConverter.InverseInstance}}"/>
-            <Button DockPanel.Dock="Right" Padding="3,0" Margin="0,3,5,3" Height="NaN" Command="{s:Action ResumeLog}" Content="{l:Loc ConsoleView_ResumeButton}" Visibility="{Binding LogPaused, Converter={x:Static s:BoolToVisibilityConverter.Instance}}"/>
+            <Button DockPanel.Dock="Right" Padding="3,0" Margin="0,3,5,3" Height="NaN" Width="75" Command="{s:Action ShowSettings}" Content="{l:Loc ConsoleView_SettingsButton}"/>
```
I still think this should use `MinWidth` rather than `Width`

danielchalmers replied:
Is it likely to be larger than 75 on other languages?
Would look inconsistent and could clip if the shell is at minimum width.

danielchalmers commented:
I have specified a minimum size and fixed enter/escape hotkey behaviour for the "Unhandled Exception", "Third Party Components", and "New Version Alert" windows, and changed the console buttons to use a minimum width instead of fixed width.

canton7 commented:
Thanks! I tweaked those button sizes back to 50px, and reduced the size of the separators a bit.

I merged it manually as https://github.com/canton7/SyncTrayzor/commit/34e1814b53bd0817cf52cc4485af6ee6fb9fa936 so I could squash things up a bit.

canton7 commented:
Hmm, this removed all of the chrome from the About screen. I'm going to revert that bit.
